### ISSUE TITLE
Sample or example?

### ISSUE THREAD

**Issue by gqqnbig** [tensorflow/tensorflow#48298]
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/experimental/preprocessing/TextVectorization

### Clear description

Lots of the description use the word _sample_. However, [Google Machine Learning Glossary](https://developers.google.com/machine-learning/glossary) does not have an entry of _sample_. It does have _example_:

> example
> One row of a dataset. An example contains one or more features and possibly a label. See also labeled example and unlabeled example.

Since this class works on dataset, I think the description is talking about examples.

Before we have samples, there must be a process of sampling. I don't think TextVectorization is doing sampling. Therefore it's unproper to call examples samples.

Should "sample" be replaced by "example"?

**Comment by MarkDaoust** [comment]
Yes. "Example" would be a slightly better fit. "Sample" is used in the sense "sample from a probability distribution", we often think of datasets as sample from a distribution, and an example is a sample of size 1.

If you send a PR I'll approve it.
